src/client.ts

```diff
@@ -6,4 +6,16 @@ const client = axios.create({
   baseURL,
 });
 
+client.interceptors.request.use((config) => {
```

Comment:
maybe create folder (httpClient, client, api etc) and replace creating client here and create a folder with interceptors